src/fu-keyring-utils.c

```diff
@@ -140,7 +140,7 @@ fu_keyring_get_release_trust_flags (AsRelease *release,
 	}
 
 	/* check we were installed correctly */
-	pki_dir = g_build_filename (SYSCONFDIR, "pki", "fwupd", NULL);
+	pki_dir = g_build_filename (CONFIGDIR, "pki", PACKAGE_NAME, NULL);
```

Comment:
This one won't work in snap, but I guess since fwupdtool will be the only one in snap it's not going to matter very much for now.

Reply:
How come it won't work with snap?

Reply:
`CONFIGDIR` is hardcoded to `/etc` rather than being able to dynamically get `$SNAP_USER_DATA/etc`.
`FU_PATH_KIND_CONFIGDIR_PKG` will return `$SNAP_USER_DATA/etc/fwupd` and this is configuring `/etc/pki/fwupd`.

I guess one really easy solution is to move this PKI directory to `/etc/fwupd/pki` instead.

Reply:
I think i'd rather just fix the bug :) Basically, we needed `FU_PATH_KIND_CONFIGDIR` rather than the `_PKG` version. I've pushed a new rebased version.